module/dice/d20-roll.mjs

```diff
@@ -60,6 +75,28 @@ export default class D20Roll extends Roll {
     return this.options.advantageMode === D20Roll.ADV_MODE.DISADVANTAGE;
   }
 
+  /* -------------------------------------------- */
+
+  /**
+   * Is this roll a critical success? Returns undefined if roll isn't evaluated.
+   * @type {boolean|void}
+   */
+  get isCritical() {
+    if ( !this._evaluated ) return undefined;
+    return this.dice[0].total >= (this.options.critical ?? 20);
```

Comment:
I think it should be possible to have a d20 roll where a critical success is not possible. If we wanted to be a bit more buttoned-up about the rules, for example, we might not highlight a nat 20 as a "critical success" for things like skill checks or saving throws for which that result has no particular meaning. 

I think for such rolls, it should be allowed that `this.options.critical` would be `null` to indicate "it is not possible to achieve a critical success" - which would require the logic to only consider cases in which `this.options.critical` has `Number.isNumeric()`